tools/coverage/docs/runLocally.md

```diff
@@ -0,0 +1,15 @@
+A pre-submit check, to see how the coverage of your local repository compare to the lastest postsubmit run, can be run locally in the following way
+
+1. download https://github.com/kubernetes/test-infra/tree/master/robots/coverage
+
+2. build the binary
+  - `go build -o coverage`
+  
+3. run the binary to download coverage profile from the latest healthy build of the given job
```

Comment:
can we have a script that effectively does steps 3-5 given the prowjob_name. So, I can do 
`JOB_NAME=<foo> ./coverage diff`

This will:
- Download the latest to BASE_PROFILE
- Run go test and save it to a temp location NEW_PROFILE
- Diff these two and return the result.